src/app/app-routing.module.ts

```diff
@@ -1,7 +1,12 @@
 import { NgModule } from '@angular/core';
 import { RouterModule, Routes } from '@angular/router';
+import { AppComponent } from './app.component';
 
-const routes: Routes = [];
+const routes: Routes = [
+  { path: '', pathMatch: 'full', component: AppComponent },
+  // { path: 'login', component: LogInComponent },
+  // { path: 'register', component: RegisterComponent }
+];
```

Comment:
remove commented out code from everywhere

Reply:
removed unnecessary comments. please review it in the new commit.